libs/components/core/src/lib/modules/scrollable-host/scrollable-host.service.ts

```diff
@@ -190,6 +204,42 @@ export class SkyScrollableHostService {
     });
   }
 
+  public watchScrollableHostClipPathChanges(
+    elementRef: ElementRef
+  ): Observable<string> {
+    if (!this.#resizeObserverSvc) {
+      throw new Error(
+        'The SkyResizeObserverService is required to use the watchScrollableHostClipPathChanges method.'
+      );
+    }
```

Comment:
Error because `this.#resizeObserverSvc` is `@Optional` and `undefined` if it wasn't provided.

Reply:
This is still technically a breaking change; the error just occurs at runtime instead of buildtime. Can we just fail silently when `#resizeObserverSvc` is undefined?

Reply:
Switched to failing silently.